lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -306,7 +307,8 @@ private class MutationExtension<T : Any>(val spec: MutationExtensionSpec<T>) : E
 
   override lateinit var data: RecyclerData<T>
 
-  internal fun setUpDragHandle(view: View) {
+  @SuppressLint("ClickableViewAccessibility")
+  fun setUpDragHandle(view: View) {
```

Comment:
Why make this public? Doesn't seem related.

Reply:
It's redundant, because the class itself is private.